src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
The variable '$gap' is used here, but it is not defined within this file. Ensure that '$gap' is defined in this file or imported from another SCSS file.